src/lib_ccx/CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 cmake_policy (SET CMP0037 NEW)
 
-set (CMAKE_C_FLAGS "-O0 -Wall -g -std=gnu99")
+if(MSVC)
+    set (CMAKE_C_FLAGS "-W3 /wd4005 /wd4996")
+else (MSVC)
+    set (CMAKE_C_FLAGS "-Wall -g -std=gnu99")
```

Comment:
removed O0 from original linux compilation, if it is unintentional please add it again, or provide reason

Reply:
@anshul1912 Doesn't -O0 mean turn off optimization? Why do we want to turn off optimization?

Reply:
So you want to turn on optimization thats a good reason for removing.